Source/Scene/GlobeDepth.js

```diff
@@ -21,26 +17,20 @@ import StencilOperation from "./StencilOperation.js";
  * @private
  */
 function GlobeDepth() {
-  this._globeColorTexture = undefined;
-  this._primitiveColorTexture = undefined;
-  this._depthStencilTexture = undefined;
-  this._globeDepthTexture = undefined;
-  this._tempGlobeDepthTexture = undefined;
   this._tempCopyDepthTexture = undefined;
 
-  this._globeColorFramebuffer = undefined;
-  this._primitiveColorFramebuffer = undefined;
```

Comment:
While you're at it could you remove the code in `Scene.js` related to `separatePrimitiveFramebuffer`?